Playmate/Utilities/Constants.h

```diff
@@ -36,6 +36,10 @@ NS_ASSUME_NONNULL_BEGIN
 + (NSString *)formatDateShort:(NSDate *)original;
 + (NSArray *)listOfSystemColors;
 
+// Info for incoming friend notifications
++ (NSString *)acceptedConfirmationStringFor:(NSString *)name;
++ (NSString *)deniedConfirmationStringFor:(NSString *)name;
```

Comment:
This will probably represent a medium but easy refactor, you can use a file for constants, such as colors, sizes of views, Images, but it'd be nice to put Strings on a separate file, strings that the user can read, in the end you would end up having a file for constants (things the user not necessarily understands or know about) and one for strings (strings the user reads and understands)